src/Components/Facility/LocationManagement.tsx

```diff
@@ -224,97 +225,120 @@ const Location = ({
   disabled,
   setShowDeletePopup,
   facilityId,
-}: LocationProps) => (
-  <div className="flex h-full w-full flex-col rounded border border-secondary-300 bg-white p-6 shadow-sm transition-all duration-200 ease-in-out hover:border-primary-400">
-    <div className="flex-1">
-      <div className="flex w-full items-start justify-between gap-2">
-        <div className="flex items-end gap-3">
-          <p className="break-all text-xl font-medium" id="view-location-name">
-            {name}
-          </p>
-          <div
-            className="mt-2 h-fit rounded-full border-2 border-primary-500 bg-primary-100 px-3 py-[3px]"
-            id="location-type"
-          >
-            <p className="text-xs font-bold text-primary-500">
-              {location_type}
+}: LocationProps) => {
+  const { loading, data } = useQuery(routes.listFacilityBeds, {
+    query: {
+      facility: facilityId,
+      location: id,
+    },
+  });
+
+  const totalBeds = data?.results.length || 0;
```

Comment:
```suggestion
  const totalBeds = data?.count ?? 0;
```

<img width="739" alt="image" src="https://github.com/user-attachments/assets/611f7b90-583f-4670-a0ae-c9e16573a70e">

@noufalrahim Use the `count` value instead of `results.length` as results only contains paginated values. Eg: If there are 50 beds, results would only contain first 10 beds (depends on the limit query param)